src/services/infra/DynamoDBService.ts

```diff
@@ -0,0 +1,72 @@
+import autoBind from "auto-bind"
+
+import { config } from "@config/config"
+
+import {
+  MessageBody,
+  SiteLaunchLambdaStatus,
+} from "@root/../microservices/site-launch/shared/types"
+
+import DynamoDBClient, { UpdateParams } from "./DynamoDBClient"
+
+export default class DynamoDBService {
+  private readonly dynamoDBClient: DynamoDBClient
+
+  private readonly TABLE_NAME: string
+
+  // todo: delete these mock after integration
```

Comment:
nit: prefer to have these tracked in an issue

Reply:
added issue here: d9fffcfc